examples/inference/HF_inference.py

```diff
@@ -95,17 +96,17 @@ def run_master(pp_ranks, args):
 
     # preparing inputs based on the model choice
     if 't5' in args.model_name:
-        inp = torch.empty(bs, seq_length, dtype=torch.long, device=device).random_(model.config.vocab_size)
+        inp = torch.empty(bs, seq_length, dtype=torch.long).random_(model.config.vocab_size)
```

Comment:
You seem to have mentioned that this works but I never understand why :)
Have you tested it under both cuda and cpu modes?